modules/models.py

```diff
@@ -184,6 +184,7 @@ def huggingface_loader(model_name):
                     'bnb_4bit_compute_dtype': eval("torch.{}".format(shared.args.compute_dtype)) if shared.args.compute_dtype in ["bfloat16", "float16", "float32"] else None,
                     'bnb_4bit_quant_type': shared.args.quant_type,
                     'bnb_4bit_use_double_quant': shared.args.use_double_quant,
+                    'disable_exllama' : shared.args.disable_exllama
```

Comment:
Does this really disable ExLlama for GPTQ? This params dict is for bitsandbytes load_in_4bit not GPTQ.

The documentation recommends

```
import torch
gptq_config = GPTQConfig(bits=4, disable_exllama=False)
model = AutoModelForCausalLM.from_pretrained("{your_username}/opt-125m-gptq", device_map="auto", quantization_config = gptq_config)
```

the question remains if by manually supplying a `GPTQConfig` object requires you to manually specify all values (bits, groupsize), or if you can just change disable_exllama.

Reply:
Btw it works for me, the arg is the same so very little change required. I really didn't want to create a new one but give it a shot.

I haven't tested the training side of it because we have another issue with training 4bit Transformers at the moment.